deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,232 @@
+{
+  "apiVersion": "kubedirector.hpe.com/v1beta1",
+  "kind": "KubeDirectorApp",
+  "metadata": {
+    "name": "datafabric610"
+  },
+  "spec": {
+    "systemdRequired": true,
+    "defaultPersistDirs": [
+      "/opt/mapr"
+    ],
+    "config": {
+      "configMeta": {
+        "storage": "60",
+        "secure": "true"
+      },
+      "roleServices": [
+        {
+          "roleID": "control-system",
+          "serviceIDs": [
+            "warden",
+	    "zookeeper",
```

Comment:
zookeeper runs on this role?

Reply:
zookeeper runs on control-system, cldb and resource-manager roles. This is similar to the optimised image that is shipped in EPIC App Store